src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+unreleased
+++++++++++
+* add korea south and korea central to preview regions
```

Comment:
@rjtsdl I think this note should go under `2.0.22` instead. Travis CI said:
`We no longer require 'unreleased' in headings. Use the appropriate version number instead.`